src/ClusterBootstrap/services/monitor/lustre-exporter.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: lustre-exporter
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: lustre-exporter
+  template:
+    metadata:
+      labels:
+        app: lustre-exporter
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/path: "/metrics"
+        prometheus.io/port: "{{cnf["lustre-exporter"]["port"]}}"
+      name: lustre-exporter
+    spec:
+      nodeSelector:
+        lustre: active
+      containers:
+      - image: dlws/lustre-exporter:v2.0.0
```

Comment:
why this image? I think you build the image below

Reply:
will change to the configurable to avoid breaking prod when there is change